python/cm/adcm_config.py

```diff
@@ -517,12 +517,13 @@ def ui_config(obj, cl):  # pylint: disable=too-many-locals
 def get_action_variant(obj, conf):
     obj_conf = {}
     if obj.config:
-        cl = ConfigLog.objects.get(obj_ref=obj.config, id=obj.config.current)
-        obj_conf = cl.config
-    for c in conf:
-        if c.type != 'variant':
-            continue
-        c.limits['source']['value'] = cm.variant.get_variant(obj, obj_conf, c.limits)
+        cl = ConfigLog.objects.filter(obj_ref=obj.config, id=obj.config.current).first()
+        if cl:
+            obj_conf = cl.config
```

Comment:
Maybe it's redundant assign and we can use `cl.config` directly where it is necessary

Reply:
Sure! Delete obj_conf